homepage/models.py

```diff
@@ -46,17 +46,20 @@ class User(models.Model):
     password = models.CharField(max_length=30)
 
 
+class Professor(models.Model):
+    name = models.CharField(max_length=100)
+
+
 class Review(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     rate = models.SmallIntegerField()
     date = models.DateTimeField(default=timezone.now)
     content = models.TextField()
     course_load = models.SmallIntegerField()
-
-
-class Professor(models.Model):
-    name = models.CharField(max_length=100)
+    likes_num = models.SmallIntegerField(default=0)
+    professor_id = models.ForeignKey(Professor, on_delete=models.CASCADE, null=True, blank=True)
+    image = models.ImageField(null=True, blank=True, upload_to="images/")

```

Comment:
I changed the name of the field to "image" instead of "attachment". Because I believe users will upload a histogram as a screenshot and not as a file, like in the forum on Facebook. Let me know if it makes sense

Reply:
I think its fine for now to allow only images. 
maybe we can add support for PDFs later (this will probably be a different class field anyways) - it is not essential anyways.